src/styles/blocks/card.scss

```diff
@@ -0,0 +1,98 @@
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  padding: 16px;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin-top: 16px;
+    margin-left: 3px;
+    margin-bottom: 0;
+    display: block;
+  }
+
+  &__title {
+    margin-top: 40px;
+    margin-bottom: 0;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    text-align: left;
+    color: #060b35;
+  }
+
+  &__code {
+    margin-top: 4px;
+    font-size: 10px;
+    line-height: 14px;
+    text-align: left;
+    color: #616070;
+  }
+
+  .review {
+    display: flex;
+    justify-content: space-between;
+    height: 16px;
+    margin-top: 16px;
+
+    &__amount {
+      display: flex;
+      flex-direction: column-reverse;
+      height: 16px;
+      margin: 0;
```

Comment:
```suggestion
```

Reply:
i can't style the positioning correctly without this line. i automatically get margin-block-start, margin-block-end